src/app/starredentries/starredentries.component.html

```diff
@@ -51,21 +62,52 @@ <h3 class="starred-header">Starred Workflows</h3>
         </div>
 
         <mat-list>
-            <mat-list-item *ngFor="let workflow of starredWorkflows; let last = last;">
-              <h3 class="spacing" mat-line>
-                <a href="/workflows/{{workflow.full_workflow_path}}">
+          <mat-list-item *ngFor="let workflow of starredWorkflows; let last = last;">
+            <h3 class="spacing" mat-line>
+              <a href="/workflows/{{workflow.full_workflow_path}}">
                 {{workflow?.organization + '/' + workflow?.repository + (workflow?.toolname ? '/' + workflow?.toolname : '')}}
-                </a>
-              </h3>
-              <span mat-line>
+              </a>
+            </h3>
+            <span mat-line>
                 <small class="spacing">Last updated {{ workflow?.lastUpdated | date }}</small>
                 <span class="spacing">{{workflow?.descriptorType | uppercase}}</span>
                 <a class="spacing" *ngIf="workflow?.provider" href="{{workflow?.providerUrl}}">{{workflow?.provider}}</a>
               </span>
-              <app-starring class="pull-right" style="width: 100px;" [workflow]="workflow" (change)="starGazersChange()"></app-starring>
-              <mat-divider></mat-divider>
-            </mat-list-item>
-          </mat-list>
+            <app-starring class="pull-right" style="width: 100px;" [workflow]="workflow" (change)="starGazersChange()"></app-starring>
+            <mat-divider></mat-divider>
+          </mat-list-item>
+        </mat-list>
+
       </div>
-  </div>
-</div>
+    </div>
+  </mat-tab>
+
+  <mat-tab label="Starred Organizations">
+    <div *ngIf="starredOrganizations">
+      <div class="container">
+        <div class="row">
+          <h3 class="starred-header">Starred Organizations</h3>
+          <mat-card class="alert alert-info starred-header" role="alert" *ngIf="starredOrganizations.length==0">
+            <mat-icon>warning</mat-icon>
+            You have no starred organizations.
+          </mat-card>
+        </div>
+
+        <mat-list>
+          <mat-list-item *ngFor="let organization of starredOrganizations; let last = last;">
+            <h3 class="spacing" mat-line>
+              <a href="/organizations/{{organization.name}}" [matTooltip]="organization?.name">{{organization.displayName}}</a>
+            </h3>
+            <span mat-line>
+            <small class="spacing">Email: <a [href]="'mailto:' + organization.email">{{ organization.email }}</a></small>
+            <small class="spacing">Website: <a [href]="organization?.link">{{ organization?.link }}</a> </small>
```

Comment:
Should this have `_target="blank"`? We're not consistent on it, but I tend to think we should.